docker/Makefile

```diff
@@ -672,6 +674,24 @@ update:
 		cat .git/refs/heads/${$(@:.log=)_branch}; \
 	fi
 
+hpp-pinocchio.checkout:
```

Comment:
Why do you need to redefine those rules. There are default rules for `checkout`, `install`, `update`,...
They only need to be overriden if the default rule does not work. I do not think this is the case here.